app/views/idv/doc_auth/welcome.html.erb

```diff
@@ -7,10 +7,21 @@
         header: t('idv.welcome.no_js_header'),
         intro: t('idv.welcome.no_js_intro', sp_name: decorated_session.sp_name || t('doc_auth.info.no_sp_name')),
       ) do %>
+
+    <% if @current_user&.decorate&.reproof_for_irs?(service_provider: @current_sp) %>
+    <%= render AlertComponent.new(
+          type: :info,
+          message: t('doc_auth.info.irs_reproofing_explanation'),
+          class: ['margin-bottom-2', 'irs-reproofing-notice'],
```

Comment:
We renamed the class from the suggestion in https://github.com/18F/identity-idp/pull/7216#discussion_r1004942291

```suggestion
          class: ['margin-bottom-2', 'usa-alert--info-important'],
```

Reply:
Changed.